include/dos_system.h

```diff
@@ -305,10 +314,34 @@ class DOS_Drive {
 	virtual bool isRemovable(void)=0;
 	virtual Bits UnMount(void)=0;
 
-	const char * GetInfo() const { return info; }
+	DosDriveType GetType() const
+	{
+		return type;
+	}
+	const char *GetInfo() const
+	{
+		return info;
+	}
+	std::string GetInfoString() const
```

Comment:
Can return a `const char *`, which is what MSG_Get returns.

Reply:
It is not exactly MSG_Get, but a combination of const char * and std::string for some cases (except virtual and unknown). If changing to const char * may have to use .c_str() for these cases.

Reply:
Ahh ok, I see the `+ info;` additions. :+1: